back/engines/commercial/analytics/app/services/analytics/query_schema.json

```diff
@@ -0,0 +1,117 @@
+{
+  "type" : "object",
+  "$schema": "http://citizenlab.co/tenant_settings_spec.json",
```

Comment:
Won't change much, but the `$schema` property is not correct here. We typically refer to the JSON spec Draft-4.